file/file_test.go

```diff
@@ -52,18 +52,4 @@ func TestFile(t *testing.T) {
 	if c.Contains(testKey) {
 		t.Errorf("contains failed: the key %s should not be exist", testKey)
 	}
-
-	c = New("./test/")
```

Comment:
Why were these tests removed?

Reply:
beause i add `mkdirall dir` when new file cachego,  they are in conflict. 😁

```go
func New(dir string) cachego.Cache {
	return &file{dir}+
+	_ = os.MkdirAll(dir, os.ModePerm)
	return &file{dir: dir}
}
```

Reply:
If you think it is unnecessary, then I will delete this logic. 😅

Reply:
Well, let's remove for now, and give this responsibility to the client, we don't know if the client has different ways to handle or control the folder.

Reply:
yes，make sense. 

done.